web-core/src/main/java/com/tutrit/controller/CarController.java

```diff
@@ -0,0 +1,14 @@
+package com.tutrit.controller;
+
+import com.tutrit.gateway.CarGateway;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class CarController {
```

Comment:
Controller is unnecessary in this module. 
Please, take another look on example at https://github.com/max-java/quickstart-multimodule-maven

Reply:
I moved it to web-ui